src/index.js

```diff
@@ -34,6 +34,26 @@ export function toggleCreator(context) {
   }
 }
 
+export function handleSumbit(func){
+  return (e) => {
+    e.preventDefault();
+
+    const form = e.target;
+    let values = {};
+
+    let children = [];
+
+    children.push(form.querySelectorAll('input, select, textarea'));
```

Comment:
I still wonder if using elements is cleaner. With the query selector, we'd need to update it if we had other things to collect info from.  But elements would do that for us, I think? Unless I'm mistaken.

In slack, you said elements includes the button? I don't think that's a problem. 

Though, I wonder if there is a way to do .elements&[name] or something like​ that. Cause we can only care about named things, yeah?